src/Pim/Bundle/CatalogBundle/Resources/config/repositories.yml

```diff
@@ -119,6 +120,14 @@ services:
         tags:
             - { name: 'pim_repository' }
 
+    pim_catalog.repository.product_unique_data:
+        class: '%pim_catalog.repository.product_unique_data.class%'
+        factory_service: doctrine.orm.entity_manager
```

Comment:
Those two methods are deprecated. Please use the new one: 
`factory: ['@doctrine.orm.entity_manager', getRepository]`

→ https://symfony.com/doc/2.7/service_container/factories.html.